puppet/modules/slave/manifests/packaging/rpm.pp

```diff
@@ -114,4 +116,10 @@
   package { ['dnf', 'dnf-plugins-core']:
     ensure => present,
   }
+
+  secure_ssh::rsync::uploader_key { 'yumstage':
```

Comment:
This means it's present on every host. That may not be a bad thing, but in the receiver setup it's limited to just OSUOSL nodes and that feels inconsistent.

The primary reason we limited it in the past is that we had sponsored machines where others had root. I don't think that's the case today so we can simplify it and allow running it on every node.

For Debian it doesn't matter that much because it's just a single machine.

Reply:
I don't know if I tackled the update for this correctly so please check it.